sysutils/pyinfra/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                pyinfra
+version             1.4
+revision            0
+
+homepage            https://pyinfra.com
+
+description         pyinfra automates infrastructure super fast at massive \
+                    scale.
+
+long_description    {*}${description}  It can be used for ad-hoc command \
+                    execution, service deployment, configuration management \
+                    and more. Core design features include super fast \
+                    execution over thousands of hosts with predictable \
+                    performance, agentless execution against \
+                    SSH/Docker/subprocess/winrm hosts, extendable with any \
+                    Python package as configured & written in standard \
+                    Python, and integrated with Docker, Vagrant/Mech & \
+                    Ansible out of the box.
+
+platforms           darwin
+license             MIT
+categories          sysutils python
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+supported_archs     noarch
+
+checksums           rmd160  3c428307414bc86d06126ddb7aeb5fbbba007cdd \
+                    sha256  3cad5ca81b4d842091f39a6e24ea8117e9c2fcb1a4abfcfb254bf1617d87dca0 \
+                    size    166876
+
+python.default_version \
+                    39
+
+depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
pyinfra contains [entry_points](https://github.com/Fizzadar/pyinfra/blob/92cd818ce72a6a1bec030932e2f11f3e9732bb77/setup.py#L107-L110), which means setuptools will be required at both build and runtime (hence it should be a lib dependency). See https://github.com/macports/macports-ports/pull/6314#issuecomment-585490282

```
entry_points={
            'console_scripts': (
                'pyinfra=pyinfra_cli.__main__:execute_pyinfra',
            ),
```

Reply:
Good point; change made